<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuseServiceSelectorDlgUI</class>
 <widget class="QWidget" name="FuseServiceSelectorDlgUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>82</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbProtocol">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>Protocol:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>leProtocol</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leProtocol">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the remote KIO protocol, e.g. &lt;span style=&quot; font-weight:600;&quot;&gt;smb&lt;/span&gt; or &lt;span style=&quot; font-weight:600;&quot;&gt;ftp&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbService">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>Service:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbServices">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the desired FUSE mount service.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
